manim/mobject/types/vectorized_mobject.py

```diff
@@ -414,6 +414,16 @@ def set_sheen_direction(self, direction, family=True):
             self.sheen_direction = direction
         return self
 
+    def rotate_sheen_direction(self, angle, axis=OUT, family=True):
```

Comment:
You should add documentation to your new function. Also type hints would be a nice addition

Reply:
So I added some docs and type hints, I am not sure if I did it correctly or not lol